ew/static/cosmetics.py

```diff
@@ -1316,6 +1316,16 @@
         style = "beautiful",
         freshness = 10,
     ),
+    EwCosmeticItem(
```

Comment:
since it's an adornable item, and since you can adorn items on your slimeoids, i wonder if you need to add new tag commands for these. it might be easier for players if you set this as a condition to the !dressslimeoid command, where the tag value is whatever you put in with !scrawl. 

i'm also thinking whether it's a good idea to use a distinct tag column or if we want to use these to change the slimeoid's name permanently. probably not, because people might lose track of their slimeoids, they could have serious attachment to those

also, this is just a scattershot idea, but what if you made dog tags always drop on death even when adorned, sort of like how military uses dog tags to identify the dead